G/GPUArrays/Compat.toml

```diff
@@ -51,3 +51,8 @@ julia = "1.0-1.3"
 ["0.6"]
 Adapt = "0.4.1-0.4.2"
 FillArrays = "0.3-0.5"
+
+["0.7"]
+Adapt = ["0.4.1-0.4"]
+FillArrays = ["0.3.0-0.3", "0.4.0-0.4", "0.5.0-0.5", "0.6.0-0.6"]
+julia = ["1.0.0-1"]
```

Comment:
@StefanKarpinski is it ok to merge PR's with this bug? Can we normalize them afterwards?

Reply:
Yes, it’s fine, we can normalize later.